docs/source/guide/rem-2-use-case.myst

```diff
@@ -0,0 +1,34 @@
+---
+jupytext:
+  text_representation:
+    extension: .myst
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.10.3
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# When should I use REM?
+
+## Advantages
+
+Readout-error mitigation is a technique that can correct the final step of a quantum algorithm: readout. 
+These errors are likely device-specific and, as such, can be mitigated specifically that way. If a
```

Comment:
I think we should assume the reader is familiar with the technique, so the first sentence might be redundant. As for the main advantages, I thought it was mainly that this technique deals with a different type of error than most mitigation techniques do, and it can be composed with other techniques were the main advantages. WDYT of something like this:

> Readout error mitigation is a technique that deals with errors that many other techniques do not. For that reason it can often be combined with existing workflows to produce better results without much modification. The technique can also accept as much information as the user has about the measurement statistics in order to generate more accurate predictions.

Reply:
I took your verbiage for this. Thanks.